### PR TITLE
fix jumbled text in popups

### PR DESCRIPTION
**Pull request by tst-basilhuffman**

### CODE REVIEW

File: earth_enterprise/src/fusion/fusionui/AssetChooser.cpp [hunk 1]
```diff
@@ -315,9 +307,10 @@ void AssetChooser::accept() {
     // here
     AssetItem* assetItem = dynamic_cast<AssetItem*>(item);
     if (assetItem != NULL) {
-        std::string temp { shortAssetName(assetItem->getAssetHandle()->getName()
-                                          .toStdString().c_str()) };
-      nameEdit->setText(temp.c_str());
+     //   std::string temp { shortAssetName(assetItem->getAssetHandle()->getName()
```
Comment (tst-ccamp) [inline earth_enterprise/src/fusion/fusionui/AssetChooser.cpp:310]: Minor: remove commented-out code

File: earth_enterprise/src/fusion/gst/gstAssetGroup.cpp [hunk 2]
```diff
@@ -18,26 +18,43 @@
...
+      kVectorAssetSuffix, kImageryAssetSuffix, kMercatorImageryAssetSuffix,
+      kTerrainAssetSuffix, kVectorProjectSuffix, kImageryProjectSuffix,
+      kMercatorImageryProjectSuffix, kTerrainProjectSuffix,
+      kDatabaseSuffix, kMapLayerSuffix, kMapProjectSuffix,
+      kMapDatabaseSuffix, kMercatorMapDatabaseSuffix, kVectorLayerSuffix
+  };
+
+  std::string saname { n };
+
+  for (const auto& elem : suffixes)
```
Comment (tst-ccamp) [inline earth_enterprise/src/fusion/gst/gstAssetGroup.cpp:37]: I'm having trouble understanding what this code is doing, so can you add some comments?

Comment (tst-basilhuffman) [reply earth_enterprise/src/fusion/gst/gstAssetGroup.cpp:46]: done

**Review by tst-ccamp — APPROVED** [review]
Code looks good to me

File: earth_enterprise/src/fusion/fusionui/AssetChooser.cpp (hunk 1 above)
Comment (tst-basilhuffman) [reply earth_enterprise/src/fusion/fusionui/AssetChooser.cpp]: done

**Review by tst-dsugar — APPROVED** [review]
Does this need to be re-targetted to the release branch?

**Pull request merged**